src/sentry/static/sentry/app/components/projectHeader/projectSelector.jsx

```diff
@@ -97,11 +95,7 @@ const ProjectSelector = createReactClass({
       }
 
       let fullName;
-      if (features.has('new-teams')) {
-        fullName = [project.name, project.slug];
-      } else {
-        fullName = [team.name, project.name, team.slug, project.slug];
-      }
+      fullName = [project.name, project.slug];
```

Comment:
Should this just be `slug`?

Reply:
I'm not sure... likely yes. But I'll skip the functional changes in this PR.